test/geometry/calibration/test_undistort.py

```diff
@@ -191,3 +191,20 @@ def test_gradcheck(self, device):
         distCoeff = torch.rand(1, 4, device=device, dtype=torch.float64)
 
         assert gradcheck(undistort_points, (points, K, distCoeff), raise_exception=True)
+
+
+class TestUndistortImage:
+    def test_shape(self, device, dtype):
+        im = torch.rand(1, 3, 5, 5, device=device, dtype=dtype)
+        K = torch.rand(3, 3, device=device, dtype=dtype)
+        distCoeff = torch.rand(4, device=device, dtype=dtype)
+
+        imu = undistort_image(im, K, distCoeff)
+        assert imu.shape == (1, 3, 5, 5)
+
+    def test_gradcheck(self, device, dtype):
+        im = torch.rand(1, 1, 15, 15, device=device, dtype=dtype, requires_grad=True)
+        K = torch.rand(3, 3, device=device, dtype=dtype)
+        distCoeff = torch.rand(4, device=device, dtype=dtype)
+
+        assert gradcheck(undistort_image, (im, K, distCoeff), raise_exception=True)
```

Comment:
can you test jit and onnx ? /cc @aniketmaurya any suggestion for onnx here

https://github.com/kornia/kornia/blob/master/test/color/test_gray.py#L91